editor/project.clj

```diff
@@ -191,7 +191,12 @@
                                :injections [(require 'editor.reveal)]
                                :dependencies [[vlaaad/reveal "1.3.280"]]}
                       :metrics {:jvm-opts ["-Ddefold.metrics=true"]}
+                      :jamm {:dependencies [[com.github.jbellis/jamm "0.4.0"]]
+                             :jvm-opts [~(str "-javaagent:"
+                                           (.replace (System/getProperty "user.home") \\ \/)
+                                           "/.m2/repository/com/github/jbellis/jamm/0.4.0/jamm-0.4.0.jar")]}
```

Comment:
Maybe there is a better way to do this? 🤔 We're kind of limited in what we can use from `project.clj`, though. Still, this should work across all platforms, I believe.